pkg/notify/elasticsearch.go

```diff
@@ -42,8 +46,36 @@ type ElasticSearch struct {
 
 // NewElasticSearch returns new ElasticSearch object
 func NewElasticSearch(c *config.Config) (Notifier, error) {
+	const AWSService = "es"
```

Comment:
consts should be declared in the headers. e,g https://github.com/infracloudio/botkube/blob/develop/pkg/config/config.go#L30

Reply:
its local to that function though